### ISSUE TITLE
ESM-style types in legacy-style types

### ISSUE THREAD

**Issue by yishn** [SAP/ui5-typescript#392]
**Describe the bug**

When using legacy-style types @sapui5/ts-types, I can see ESM-style types in the code, e.g. in `sap.ui.core.d.ts` there are a bunch of `declare module ...`.

**Expected behavior**

I expect to see namespaces instead of modules.

**Comment by codeworrior** [comment]
This is not a bug. There are APIs in UI5 that are not exposed via a global name. These APIs are declared in module syntax and can only be accessed via imports ni TypeScript. In the JavaScript world of UI5, this corresponds to APIs that can only bi accessed via an AMD dependency and not via a global name.

Examples:
* https://openui5.hana.ondemand.com/api/module:sap/base/Log
* https://openui5.hana.ondemand.com/api/module:sap/base/i18n/ResourceBundle
* basically anything under `sap/base/`, but also a few others...
